finality-aleph/src/network/manager/discovery.rs

```diff
@@ -348,6 +352,38 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn broadcasts_when_clueless_modulo_3() {
+        let (mut discovery, mut handlers, _) = build_number(3).await;
+        let handler = &mut handlers[0];
+        let mut messages = discovery.discover_authorities(handler);
+        assert_eq!(messages.len(), 1);
+        let message = messages.pop().unwrap();
+        assert_eq!(
+            message,
+            (
+                DiscoveryMessage::AuthenticationBroadcast(handler.authentication().unwrap()),
+                DataCommand::Broadcast
+            )
+        );
+    }
+
+    #[tokio::test]
+    async fn broadcasts_when_clueless_just_2() {
```

Comment:
Pls add one more test when it is not multiple of 3 and is bigger than 3

Reply:
As above, although this already existed.